app/Models/BillHistoryTimestamps.php

```diff
@@ -2,10 +2,56 @@
 
 namespace App\Models;
 
+use App\Models\LegiScan\Bill;
+use App\Notifications\BillHasProgressed;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Notification;
 
 class BillHistoryTimestamps extends Model
```

Comment:
We really REALLY need to find out how to break out out of putting namespaces into class names. Namespaces exist to bucket code topically for human brain consumption and organization. They also `KeepUsFromHavingModelAndClassNamesLikeThis`. When I walk into a Laravel app and there's 100 Models in one folder and 10 of them have the same prefix and another 12 of the them have the same other prefix, I just want to tear my hair out. Make some folders and organize things lexically into them.

This is either

`Models\Bill\HistoryTimestamp`
or
`Models\BillHistory\Timestamp`

Also, Model names are never plural.

Reply:
Ok, I will work on this. The reason for the plural was because I have 2 timestamp columns (and not because the table had many rows) ... but I get what you are saying! Making the fixes now.